src/cycles/johnson.jl

```diff
@@ -55,7 +55,7 @@ end
 ```
 type JohnsonVisitor{T<:Integer} <: Visitor{T}
     stack::Vector{T}
-    blocked::BitArray
+    blocked::BitArray{1}

```

Comment:
Nice, but we should just call this `BitVector`.

Reply:
ups :man_facepalming: